Dockerfile

```diff
@@ -85,6 +85,8 @@ RUN curl https://raw.githubusercontent.com/aws/aws-sdk-net/master/sdk/src/Core/e
     && cp /etc/endpoints_new.json $(python${PYTHON_VERSION} -c "import boto; print('%s/endpoints.json' % boto.__path__[0])")
 
 # Contain ec2.py dynamic inventory from https://raw.githubusercontent.com/ansible/ansible/devel/contrib/inventory/ec2.py
-COPY files/ansible/ /etc/ansible/
+COPY files/ansible/ansible.cfg /etc/ansible/ansible.cfg
+COPY files/ansible/hosts /etc/ansible/hosts-template/
+RUN mkdir /etc/ansible/hosts
```

Comment:
The `/etc/ansible/hosts` directory is automatically created by ansible setup so I think you can remove this part.
Maybe simplify by just renaming our `files/ansible/hosts` directory in git to `files/ansible/hosts-template`

Reply:
`/etc/ansible/hosts` is needed by the bash script when we `mv` / `envsub` into this directory (so before running ansible).

Reply:
I mean this directory should be created by the install of ansible : https://github.com/cycloidio/docker-cycloid-toolkit/blob/develop/Dockerfile#L52

Reply:
It's not ! (I removed the line to test)

Reply:
indeed it seems to be created by the Debian package, not the pip install.
I would go for a rename of our git directory `files/ansible/hosts` -> `files/ansible/hosts-template/`

and about the  `/etc/ansible/hosts` up to you to let the mkdir, put it into the script or create a `files/ansible/hosts/.gitkeep` file into our git to automatically create this file